packages/loader/container-loader/src/container.ts

```diff
@@ -1388,38 +1451,23 @@ export class Container
 				// Start prefetch, but not set opsBeforeReturnP - boot is not blocked by it!
 				// eslint-disable-next-line @typescript-eslint/no-floating-promises
 				this.attachDeltaManagerOpHandler(
-					attributes,
+					dmAttributes,
 					loadMode.deltaConnection !== "none" ? "all" : "none",
 				);
 				break;
 			case "cached":
-				opsBeforeReturnP = this.attachDeltaManagerOpHandler(attributes, "cached");
+				opsBeforeReturnP = this.attachDeltaManagerOpHandler(dmAttributes, "cached");
 				break;
 			case "all":
-				opsBeforeReturnP = this.attachDeltaManagerOpHandler(attributes, "all");
+				opsBeforeReturnP = this.attachDeltaManagerOpHandler(dmAttributes, "all");
 				break;
 			default:
 				unreachableCase(loadMode.opsBeforeReturn);
 		}
 
 		// ...load in the existing quorum
 		// Initialize the protocol handler
-		if (pendingLocalState === undefined) {
-			await this.initializeProtocolStateFromSnapshot(
-				attributes,
-				this.storageAdapter,
-				snapshot,
-			);
-		} else {
-			this.initializeProtocolState(
-				attributes,
-				{
-					members: pendingLocalState.protocol.members,
-					proposals: pendingLocalState.protocol.proposals,
-					values: pendingLocalState.protocol.values,
-				}, // pending IQuorumSnapshot
-			);
-		}
+		await this.initializeProtocolStateFromSnapshot(attributes, this.storageAdapter, snapshot);
 
```

Comment:
why isn't this dmAttributes?

Reply:
This needs the sequence number from the snapshot since we will replay quorum ops. The sequence number should be the same as the one in dmAttributes after saved ops have been replayed.